vividus-util/src/main/java/org/vividus/util/property/IPropertyMapper.java

```diff
@@ -43,6 +44,21 @@
      */
     <T> PropertyMappedCollection<T> readValues(String propertyPrefix, Class<T> valueType) throws IOException;
 
+    /**
+     * Maps properties with specified prefix to the collection with values representing objects created
+     * using property names and their values. If the base prefix is specified then every property property
+     * found by the base prefix will be added to the properties found by the main prefix if it's missing
+     * there
+     * @param <T> type of resulting objects
+     * @param propertyPrefix Prefix of properties
+     * @param basePropertyPrefix Prefix of base properties
+     * @param valueType Collection value type
+     * @return collection with values representing objects created using property names and their values
+     * @throws IOException if any error is occurred during mapping of properties to objects
+     */
+    <T> PropertyMappedCollection<T> readValues(String propertyPrefix, Optional<String> basePropertyPrefix,
```

Comment:
```suggestion
    <T> PropertyMappedCollection<T> readValues(String propertyPrefix, String basePropertyPrefix,
```
If I don't need `basePropertyPrefix`, then I'll used another method